CONTRIBUTING.md

```diff
@@ -1 +1,25 @@
 # Developer Guide
+
+Thank you for taking the time out to contribute to project Theia!
+Project Theia follows the [Antrea Developer Guide]
+
+## Notice
+
+Please check out the [Theia document](docs/theia.md) for Theia installation and usage.
```

Comment:
theia.md is just for the theia CLI tool.

Probably change to: [Theia documents](docs/)

Reply:
Make sense. Fixed.